### PR TITLE
Feature/create html and css

### LINKED ISSUE #5: Add Dashboard
Use the scripts.js file to add information to the DOM. This JS file should call methods from your classes to retrieve data. There should not be any DOM manipulation within your class files.

- As a user, I should be able to view a list of all recipes.
- As a user, I should be able to click on a recipe to view more information including directions, ingredients needed, and total cost.
- As a user, I should be able to filter recipes by a tag. (Extension option: by multiple tags)
- As a user, I should be able to search recipes by their name. (Extension option: by name or ingredients)

### PR DESCRIPTION
**Pull request by aominhlong**
What's this PR do?
- create HTML and CSS skeleton
- be able to view a list of all recipes.
- be able to click on a recipe to view more information including directions, ingredients needed, and total cost.
- be able to filter recipes by a tag. (Extension option: by multiple tags)
- be able to search recipes by their name. (Extension option: by name or ingredients)
- refactored page toggle and remove unnecessary code

Where should the reviewer start?
- scripts.js
- index.html
- styles.css

How should this be manually tested?
1. Run `npm start` when in the main directory
2. Go to `http://localhost:8080/`

Any background context you want to provide?
N/A

What are the relevant tickets?
resolve #5 

Screenshots (if appropriate)
N/A

Questions:
None

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -2,7 +2,205 @@ import './styles.css';
...
+  foodImagesSection.classList.add("hidden");
+  allRecipesSection.classList.add("hidden");
+  favoriteRecipeArea.classList.add("hidden")
+  viewAllButton.classList.add("hidden");
+  displayRecipeSection.classList.add("hidden")
+  displayRecipeSection.innerHTML = ''
+  filteredByTagArea.classList.remove("hidden")
+}
+
+function populateRecipeArea() {
```
Comment (mattruder) [inline src/scripts.js:120]: this function can likely be reduced into helper functions!

File: src/styles.css [hunk 2]
```diff
@@ -1,3 +1,73 @@
...
+
 body {
-  background-color: blueviolet;
-}
\ No newline at end of file
+  height: 100vh;
+  margin: 0;
+}
+
+.food-images-section {
```
Comment (nhodnett) [inline src/styles.css:34]: Can we combine these food images sections, (line 17)?

File: src/styles.css [hunk 3]
```diff
@@ -1,3 +1,73 @@
+* {
```
Comment (mattruder) [inline src/styles.css:1]: lets work on refactoring our CSS when the time comes

File: src/scripts.js [hunk 4]
```diff
@@ -2,7 +2,205 @@ import './styles.css';
...
+  foodImagesSection.innerHTML = ''
+  foodImagesSection.classList.add("hidden");
+  allRecipesSection.classList.add("hidden");
+  viewAllButton.classList.add("hidden");
+  displayRecipeSection.classList.add("hidden");
+  displayRecipeSection.innerHTML = '';
+  filteredByTagArea.innerHTML= "";
+  favoriteRecipeArea.classList.remove("hidden")
+}
+
```
Comment (nhodnett) [inline src/scripts.js:157]: Nice use of helper functions crew. We'll see if we can use more on final refactor.

File: src/scripts.js [hunk 5]
```diff
@@ -2,7 +2,205 @@ import './styles.css';
...
+recipeRepo.data.forEach((recipe) => {
+  allRecipeList.innerHTML += `<li class="recipe-in-list" id=${recipe.id}>${recipe.name}</li>`
+})
+}
+
+const searchBtn = document.querySelector('.search-btn')
+
+searchBtn.addEventListener('click', searchRecipe)
+
+function searchRecipe() {
```
Comment (nhodnett) [inline src/scripts.js:80]: Need to check indentation on this function.

**Review by nhodnett — COMMENTED** [review]
Nice work crew! We're getting there!

**Pull request merged**